skops/hub_utils/_hf_hub.py

```diff
@@ -402,7 +402,14 @@ def update_env(
     -------
     None
     """
-    pass
+
```

Comment:
```suggestion
```

Also, can we remove the "Returns" section from the docstring, since nothing is returned?

Reply:
I just did!